templates/shared/parquet-table-rowcount.sql

```diff
@@ -14,6 +14,6 @@
 
 -- Query Parquet table in Impala
 USE {{ conf.staging_database.name }};
-INVALIDATE METADATA {{ table.destination.name }}_parquet;
-SELECT COUNT(*) FROM {{ table.destination.name }}_parquet;
+INVALIDATE METADATA {{ table.destination.name }}{% if conf.user_defined is defined and conf.user_defined.parquet_suffix is defined %}{{ conf.user_defined.parquet_suffix }}{% endif %};
```

Comment:
Can you give an example of how this `user_defined` block is used and what it's for?

Reply:
This raises a flag because it will break existing scripts that are expecting the `_parquet` suffix